src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -25,9 +28,16 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
     return 'Get the number of Microsoft Teams unique users by device type';
   }
 
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.period = typeof args.options.period !== 'undefined';
```

Comment:
Since period is required, this value will be always `true`, which doesn't add much value. Instead, we should track the actual value specified by the user, which should be perfectly fine since it doesn't contain any PII.

Reply:
Sure, Could you please let me know how to do that?

Reply:
```ts
telemetryProps.period = args.options.period;
```

Reply:
Done. This is fixed. All the feedback points are fixed.